frontend/src/styles.css

```diff
@@ -111,15 +111,15 @@
     --color-1-lightest             : hsla(34, 60%, 97%, 1);
     --color-1-lighter              : hsla(34, 60%, 77%, 1);
     --color-1-light                : hsla(34, 60%, 67%, 1);
-    --color-1-dark                 : hsla(34, 60%, 47%, 1);
+    --color-1-dark                 : rgb(30, 30, 35);
```

Comment:
Maybe keep this as hsla for consistency? The reason being is that their all the same as either `color-1` or `color-1`, with different *light*s - different shades.